frontend/app/templates/work_packages/tabs/activity.html

```diff
@@ -10,12 +10,14 @@
           <h3 class="activity-date"
               ng-if="currentDate !== (activities[$index-1].props.createdAt | date:'longDate')"
               ng-bind="currentDate"/>
-          <user-activity work-package="workPackage"
-                         activity="activity"
-                         activity-no="activityNo"
-                         is-initial="isInitial"
-                         input-element-id="'add-comment-text'">
-          </user-activity>
+            <activity-entry work-package="workPackage"
+                           activity="activity"
+                           activity-no="activityNo"
+                           is-initial="isInitial"
```

Comment:
Hint: `isInitial` is calculated as `activityNo == 1` and influences the correct display of the first WP activity ("was created" and does not display changes)...

I know it is unlikely, but I could imagine some commit referencing a WP before it is created:

* because of bad timestamps in VCS
* because of unintentional mentioning ("see GitHub PR #123 on that unrelated repository")

That could make a revision the "initial" activity, couldn't it?